python/doc/theory/reliability_sensitivity/taylor_moments.rst

```diff
@@ -1,144 +1,106 @@
-   is composed by the second order derivative of the output variable
-   towards the :math:`i^\textrm{th}` and :math:`j^\textrm{th}`
-   components of :math:`\ux` taken around :math:`\ux = \muX`. It yields
-   to:
-   :math:`\left( \nabla^2 h(\muX,\muX) \right)_{ij} =\left( \frac{\partial^2 h(\ux,\ux)}{\partial x^i \partial x^j}\right)_{\ux\: =\: \muX}`
+    \Cov \uX = (c_{ij})_{ij}
 
--  :math:`<,>` is a scalar product between two vectors.
 
-| **Approximation at the order 1 - Case** :math:`n_Y=1`
+where:
 
-  .. math::
+.. math::
 
-      \Expect{Y} = h(\muX)
+    c_{ij} = \Expect{\left(X_i - \Expect{X_i}\right)\left(X_j - \Expect{X_j} \right)}
 
 
-  .. math::
+with :math:`c_{ii} = \sigma_i^2`.
 
-      \Var{Y} = \sum_{i,j=1}^{n_X} \frac{\partial h(\muX)}{\partial X^i} . \frac{\partial h(\muX)}{\partial X^j}.(\Cov \uX)_{ij}
+**Case 1:** :math:`Y \in \Rset` , :math:`q=1`:
 
-| **Approximation at the order 2 - Case** :math:`n_Y=1`
+The second order Taylor expansion of :math:`h` around :math:`\ux = \vect{\mu}` is written as:
 
-  .. math::
+.. math::
 
-     \begin{aligned}
-         \begin{split}
-           \Expect{Y} = h(\muX) + \frac{1}{2}. \sum_{i,j=1}^{n_X} \frac{\partial^2 h(\muX,\muX)}{\partial x^i \partial x^j} . (\Cov \uX)_{ij}
-         \end{split}
-       \end{aligned}
+    Y = h(\vect{\mu}) + \sum_{i = 1}^d \left( \frac{\partial h}{\partial x_i }\right)(\vect{\mu})(X_i-\mu_i)
+    + \frac{1}{2} \sum_{i,j = 1}^d \left(\frac{\partial^2 h}{\partial x_i \partial x_j}\right)(\vect{\mu})
+    (X_i-\mu_i)(X_j-\mu_j) + o(||\uX||^2)
 
-| The decomposition of the variance at the order 2 is not implemented.
-  It requires both the knowledge of higher order derivatives of the model
-  and the knowledge of moments of order strictly greater than 2 of the pdf.
 
-| **Case** :math:`n_Y>1`
+If we use the first order expansion, we get:
 
+  .. math::
 
-| The perturbation approach can be developed at different orders
-  from the Taylor decomposition of the random vector :math:`\uY`. As
-  :math:`\uY=h(\uX)`, the Taylor decomposition around :math:`\ux = \muX`
-  at the second order yields to:
+       \begin{array}{lcl}
+            \Expect{Y} & \simeq & h(\vect{\mu})\\
+            \Var{Y} & \simeq & \sum_{i=1}^{d} \sigma_i^2 \left[\left(\frac{\partial h}{\partial x_i}
+            \right)(\vect{\mu}) \right]^2
+       \end{array}
 
-  .. math::
 
-      \uY = h(\muX) + <\underline{\underline{\nabla}}h(\muX) , \: \uX - \muX> + \frac{1}{2}<<\underline{\underline{\underline{\nabla }}}^2 h(\muX,\: \underline{\mu}_{\:X}),\: \uX - \muX>,\: \uX - \muX> + o(\Cov \uX)
+If we use the second order expansion, we get:
 
-where:
+  .. math::
 
--  :math:`\muX = \Expect{\uX}` is the vector of the input variables at
-   the mean values of each component.
+       \Expect{Y}  \simeq h (\vect{\mu}) + \frac{1}{2} \sum_{i,j=1}^{d} c_{ij}\left(\frac{\partial^2 h}
+       {\partial x_i \partial x_j}\right)(\vect{\mu})
```

Comment:
```suggestion
       {\partial x_i \partial x_j}\right)(\vect{\mu}).
```

Reply:
ok

Reply:
The equation of the first order variance in the case where the output is scalar is not correct.

The current equation is:

![image](https://github.com/openturns/openturns/assets/31351465/499ff380-3c56-4df8-9f7a-5f2b63c9692d)

This is valid only in the special case where the input are independent.

The (almost) correct equation is:

![image](https://github.com/openturns/openturns/assets/31351465/cb90fca0-e417-4cf2-b34a-6261f99c21ab)

But the previous equation involves derivatives with respect to random variables (as implied by the use of upcase letters), which has to be fixed.